docs/source/conf.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2017-2020 Contributors to the Pyro project.
```

Comment:
Let's just use "Copyright Contributors to the Pyro project" as recommended by Steve Winslow

Reply:
Okay, done.